### PR TITLE
[wip]Cleaning up the theme .scss files 

### PR DESCRIPTION
**Pull request by Ijin08**
Cleaning up the light and dark theme files to hopefully move away from global component specific variables.

### CODE REVIEW

File: public/sass/_variables.dark.scss [hunk 1]
```diff
@@ -21,18 +21,17 @@ $green-shade: #23843b;
 // -------------------------
 $black: #000;
 $dark-1: #141414;
+$dark-1-5: #161719;
```
Comment (dprokop) [inline public/sass/_variables.dark.scss]: Hm, not liking it much. Can we rescale the palette so that 1-5 becomes 2?

**Review by dprokop — COMMENTED** [review]
Great progress so far @Ijin08! Mind #15404. 

I think you should be the first to merge, and then we will adjust my PR if it's gonna be accepted.

File: public/sass/_variables.dark.scss [hunk 2]
```diff
@@ -100,27 +99,26 @@ $external-link-color: $sapphire-light;
...
-$abbr-border-color: $gray-3 !default;
+$abbr-border-color: $gray-2 !default;
 $text-muted: $text-color-weak;
 
 $hr-border-color: $dark-4;
 
 // Panel
 // -------------------------
-$panel-bg: #212124;
+$panel-bg: $dark-3;
```
Comment (torkelo) [inline public/sass/_variables.dark.scss]: this change does not feel great, makes panels a bit brighter and grayer and dashboard settings aside is affected.

File: public/sass/_variables.dark.scss [hunk 3]
```diff
@@ -376,7 +373,7 @@ $panel-editor-tabs-line-color: #e3e3e3;
 $panel-editor-viz-item-bg-hover: darken($sapphire-base, 46%);
 
 $panel-options-group-border: none;
-$panel-options-group-header-bg: $gray-blue;
+$panel-options-group-header-bg: $dark-3;
```
Comment (torkelo) [inline public/sass/_variables.dark.scss]: noooo, so nice with that gray blue, it gives the dark theme some slight and much needed saturation

File: public/sass/_variables.light.scss [hunk 4]
```diff
@@ -265,15 +262,15 @@ $success-text-color: lighten($green, 10%);
...
+$alert-info-bg: linear-gradient(100deg, $sapphire-base, $sapphire-shade);
 
 // popover
 $popover-bg: $page-bg;
 $popover-color: $text-color;
 $popover-border-color: $gray-5;
 $popover-shadow: 0 0 20px $white;
 
-$popover-help-bg: $sapphire-base;
+$popover-help-bg: $btn-secondary-bg;
```
Comment (torkelo) [inline public/sass/_variables.light.scss]: any reason this is using the btn variable ?

File: public/sass/_variables.dark.scss [hunk 5]
```diff
@@ -195,7 +192,7 @@ $input-box-shadow: inset 1px 0px 0.3rem 0px rgba(150, 150, 150, 0.1);
 $input-border-focus: $input-border-color;
 $input-box-shadow-focus: rgba(102, 175, 233, 0.6);
 $input-color-placeholder: $gray-1 !default;
-$input-label-bg: $gray-blue;
+$input-label-bg: $dark-3;
```
Comment (torkelo) [inline public/sass/_variables.dark.scss]: any reason for removing the gray blue?

**Review by torkelo — CHANGES_REQUESTED** [review]
looks good, but personally I feel the dark theme is worse off without the dark gray with a touch of blue that was used in a few places. I can understand that it is a problem to have many shades of gray and only one with a touch of blue, so not sure what the solution is or how much of a problem it really is. 

Also the panel background change I feel is also not great.

File: public/sass/_variables.dark.scss (hunk 1 above)
Comment (Ijin08) [reply public/sass/_variables.dark.scss]: Of course, this just wip. Probalbly gone name after lightness.

File: public/sass/_variables.dark.scss (hunk 2 above)
Comment (Ijin08) [reply public/sass/_variables.dark.scss]: yes, I tried to merge dark-3. panel-bg and blue-gray into one color. Might have become to much compromising.

File: public/sass/_variables.dark.scss (hunk 3 above)
Comment (Ijin08) [reply public/sass/_variables.dark.scss]: I agree with the saturation. But I felt panel-bg and gray-blue were so close. It got a bit weird when using variables.

File: public/sass/_variables.light.scss (hunk 4 above)
Comment (Ijin08) [reply public/sass/_variables.light.scss]: Hmm, no. Not sure what I was thinking there.

File: public/sass/_variables.dark.scss (hunk 5 above)
Comment (Ijin08) [reply public/sass/_variables.dark.scss]: yes, I tried to merge dark-3. panel-bg and blue-gray into one color. Might have become too much compromising. Felt that panel-bg and blue-gray were to close to each other.

**Comment by torkelo** [comment]
Sorry for having to revert some of the color changes and simplifications to the theme. Just feel a bit stressed now with v6 and GrafanaCon and with so many things going on and do not want to make any big color changes right now.

**Comment by dprokop** [comment]
@Ijin08 would you mind syncing you changes with `GrafanaTheme` interface and variable file templates in https://github.com/grafana/grafana/pull/15404 ?

**Pull request merged**